src/test/OrbitSvgTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace PlanetApp\Test;
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Util\Exception;
+use PlanetApp\OrbitSvg;
+
+class OrbitSvgTest extends TestCase
+{
+    public function testSuccessGenerateOrbitSvg()
+    {
+        $planetName = 'earth';
+        $planetRadius = '10';
+        $planetPosition = '100';
+        $orbitRadius = '50';
```

Comment:
these numbers are in quotes for no reason

Reply:
file removed